index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Porfolio Website</title>
+  </head>
+  <body>
+    <header id="nav-bar">
+      <div id="logo">
+        <h4>Said Laasri</h4>
```

Comment:
- [ ] Please the logo has a link `<a>` tag instead of `<h4>`. This is because the logo for the company is similar to the name of the company. So for the same reason, it would make sense to hyperlink a key person's name or company's name. this makes your job looks more professional.

Reply:
@OLIPLICHE Thank you for reviewing my project, I will make the necessary corrections as suggested